### DISCUSSION TITLE
Plugin locales in Docker version

### DISCUSSION THREAD

**Discussion by Akhun-Delar** [pretix/pretix#3271] [Q&A] [answered]
Hey all,

as explained here (https://github.com/pretix/pretix-sepadebit/issues/32), localizations/translations for plugins installed on Docker do not seem to work (not only sepadebit) - at least for me. This is my Dockerfile, am I missing something? 

```
FROM pretix/standalone:latest
USER root
RUN pip3 install pretix-sepadebit pretix-passbook pretix-pages pretix-adyen
USER pretixuser
RUN cd /pretix/src && make production
```
Any ideas?

**Comment by raphaelm** [comment, marked as answer]
There is something weird in our build process of most plugins, the .whl files on PyPI all lack their .mo files. Changing

```
        - python setup.py sdist bdist_wheel
```
to
```
        - python setup.py sdist
        - python setup.py bdist_wheel
```
helps, although I don't get why. We need to re-do the build process anyways soon, I'll try to fit this in before the next release cycle.

**Reply by raphaelm** [reply]
I've published a version 2.4.0.post1 of the sepadebeit plugin, can you try if it's better with that one?

**Reply by Akhun-Delar** [reply]
Works perfectly with that version!
